src/pages/loginPage.ts

```diff
@@ -0,0 +1,28 @@
+import { Page } from "@playwright/test";
+export default class LoginPage {
+
+    constructor(public page: Page) { }
+
+    async login(email: string, password: string) {
+        await this.enterEmail(email);
+        await this.enterLoginPassword(password);
+        await this.clickLoginBtn();
+    }
+
+    async enterEmail(emailaddress: string) {
+        await this.page.locator("input[name='email']")

```

Comment:
locators should be moved to a separate variables and then used in a await this.page.locator()

Reply:
Done